orchagent/aclorch.cpp

```diff
@@ -2120,6 +2120,46 @@ void AclOrch::doAclRuleTask(Consumer &consumer)
     }
 }
 
+void AclOrch::processPendingPorts()
+{
```

Comment:
Is it more effective to just go through the pending list (table.pendingPortSet)?